src/bin/help.ts

```diff
@@ -0,0 +1,230 @@
+import chalk from "chalk";
+
+import { logLine } from "../shared/cli/lines.js";
+
+export function logHelpText(version: string): void {
+	logLine();
+
+	logLine(chalk.cyan(`✨ template-typescript-node-package v${version} ✨`));
+
+	logLine(
+		chalk.cyan(
+			`
+      A quickstart-friendly TypeScript template with comprehensive formatting, 
+      linting, releases, testing, and other great tooling built-in.
+      `,
+		),
+	);
+
+	logLine();
+
+	logLine(chalk.black.bgGreenBright("Core options:"));
```

Comment:
[Praise] I do really like how the sections are given these headers. Along with the blank lines & spacing, it feels very clean. Nice! 🧹